COPYING

```diff
@@ -0,0 +1,19 @@
+Copyright 2018 The Ontario Institute for Cancer Research
```

Comment:
I think we usually do our copyright as `Genome Sequence Informatics`, but Morgan would be the person to confirm that with.

Reply:
This looks like the MIT license to me; is there a particular reason to not call it that?